packages/nextjs/app/configure/_components/DownloadContracts.tsx

```diff
@@ -1,7 +1,7 @@
 "use client";
 
 import { useProvider } from "@starknet-react/core";
-import React, { useState } from "react";
+import React, { useCallback, useState } from "react";
```

Comment:
just wondering if we can remove "import React" ? wdyt?

Reply:
good point, yes, we can